recipe/meta.yaml

```
@@ -60,3 +60,4 @@ extra:
     - ocefpaf
     - ericmjl
     - ngam
+    - ehfd
```

if people want, you could also add the jaxlib team here
```suggestion
    - ehfd
    - conda-forge/jaxlib
```